pkg/scheduler/framework/plugins/imagelocality/image_locality.go

```diff
@@ -79,14 +79,15 @@ func New(_ runtime.Object, h framework.FrameworkHandle) (framework.Plugin, error
 
 // calculatePriority returns the priority of a node. Given the sumScores of requested images on the node, the node's
 // priority is obtained by scaling the maximum priority value with a ratio proportional to the sumScores.
-func calculatePriority(sumScores int64) int64 {
+func calculatePriority(sumScores int64, sizeContainers int) int64 {
+	maxThresholds := maxThreshold * int64(sizeContainers)
```

Comment:
nit: s/maxThreshold/maxContainerThreshold

Reply:
done

Reply:
I was suggesting to change the const name `maxThreshold`

Reply:
done